sdk/keyvault/keyvault-keys/src/cryptographyClient.ts

```diff
@@ -125,6 +130,13 @@ export class CryptographyClient {
         value: key
       };
     }
+
+    if (pipelineOptions.remoteOnly) {
```

Comment:
Unfortunately this has always been named `pipelineOptions`. I'd like to change this to `options` - possible? Any concerns with renaming the param?